docs/es/week03/03-1.md

```diff
@@ -163,7 +163,7 @@ La otra apliación útil es en la detección de patrones en imágenes. Usualment
 <center><img src="{{site.baseurl}}/images/week03/03-1/MotifImage.png" alt="Network" style="zoom:35%;" /><br>
 Fig. 8 Detección de patrones en imágenes</center>
 
-También es importante que nuestro método con "plantillas" sea invariante al desplazamiento (al desplazar la entrada, la salida--la letra detectada--no debería cambiar). Esto se puede lograr con la transformación de la reutilización de parámetros (*weight sharing*). Como se aprecia en la figura 9, al cambiar la ubicación de "D", aún podemos detectar los patrones de esquinas pese a que fueron desplazadas. Al sumar los patrones, se activará la detección de "D".
+También es importante que nuestro método con "plantillas" sea invariante al desplazamiento (al desplazar la entrada, la salida&mdash;la letra detectada&mdash;no debería cambiar). Esto se puede lograr con la transformación de la reutilización de parámetros (*weight sharing*). Como se aprecia en la figura 9, al cambiar la ubicación de "D", aún podemos detectar los patrones de esquinas pese a que fueron desplazadas. Al sumar los patrones, se activará la detección de "D".

```

Comment:
error: "&mdash" should be changed back to "--"

Reply:
I changed them back to "--"